packages/block-library/src/post-link/index.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * WordPress dependencies
+ */
+import { link as icon } from '@wordpress/icons';
```

Comment:
We would need a new icon here. @jasmussen, @jameskoster  or any designer that finds some time 😄